src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -2,13 +2,14 @@ package io.iohk.ethereum.ledger
 
 import io.iohk.ethereum.db.storage.NodeStorage
 import io.iohk.ethereum.domain._
+import io.iohk.ethereum.network.p2p.messages.PV63.Receipt
```

Comment:
I don't like the fact that we're using network classes here. Can we create `Receipt` class in `domain` or `ledger` (whichever you think is better).